app/ui.R

```diff
@@ -121,18 +121,12 @@ ui <- fluidPage(
           checkboxInput('has_vaccine', div("I have received at least one dose of a COVID-19 vaccine")),
           conditionalPanel(
             condition = "input.has_vaccine == true",
-            radioButtons('vaccine', "Which of the available COVID-19 vaccines did you receive?",
+            radioButtons('vaccine', "Which of the available COVID-19 vaccines did you receive most recently?",
```

Comment:
Should we add ", including the booster?" to this question as well?

Reply:
I can see that this context could be helpful. @mollybair since you have a fresh pair of eyes, what do you think about the wording?